test/assert.js

```diff
@@ -130,10 +235,31 @@ test('.is()', t => {
 	}, {
 		assertion: 'is',
 		message: 'my message',
-		operator: '===',
 		values: [
 			{label: 'Actual:', formatted: /foo/},
-			{label: 'Must be strictly equal to:', formatted: /42/}
+			{label: 'Must be strictly equal (using Object.is) to:', formatted: /42/}
+		]
+	});
+
+	failsWith(t, () => {
+		assertions.is(0, -0, 'my message');
+	}, {
+		assertion: 'is',
+		message: 'my message',
+		values: [
+			{label: 'Actual:', formatted: /0/},
+			{label: 'Must be strictly equal (using Object.is) to:', formatted: /0/}
```

Comment:
I suspect this outputs `0`, rather than `-0`? https://github.com/avajs/ava/pull/1341 would fix, but I think we should work around this in this branch too.